junebug/logging_service.py

```diff
@@ -72,13 +70,13 @@ def _log_to_file(self, event):
             data['instance'] = repr(failure.value)
             data['stack'] = failure.stack
 
-        data['message'] = log.textFromEventDict(event)
-
         self.logfile.write(json.dumps(data))
 
     def __call__(self, event):
         if self.log_context_sentinel in event:
             return
+        if self.worker_id not in event.get('system', ''):
+            return
```

Comment:
We could, I'm just not sure that I see a case where those two statements could be different, besides if there is a `,` in `worker_id`, which isn't possible.